pkg/asset/manifests/operators.go

```diff
@@ -166,9 +166,13 @@ func (m *Manifests) generateBootKubeManifests(dependencies asset.Parents) []*ass
 		rootCA,
 	)
 
-	etcdEndpointHostnames := make([]string, *installConfig.Config.ControlPlane.Replicas)
+	etcdEndpointHostnames := make([]string, *installConfig.Config.ControlPlane.Replicas+1)
+	etcdEndpointHostnames[0] = "etcd-bootstrap"
 	for i := range etcdEndpointHostnames {
-		etcdEndpointHostnames[i] = fmt.Sprintf("etcd-%d", i)
+		if i == 0 {
+			continue
+		}
+		etcdEndpointHostnames[i] = fmt.Sprintf("etcd-%d", i-1)
```

Comment:
nit: I think a more idiomatic way to write this would be something like:

```go
for i, v := range etcdEndpointHostnames[1:] {
     v = fmt.Sprintf("etcd-%d", i-1)
}
```